--- a/tools/depends/target/ffmpeg/CMakeLists.txt	2020-10-31 16:19:36.000000000 -0400
+++ b/tools/depends/target/ffmpeg/CMakeLists.txt	2020-11-03 09:04:17.785889237 -0500
@@ -22,7 +22,7 @@ endif()
   string(CONCAT CMAKE_C_FLAGS ${CMAKE_C_FLAGS} " -I/opt/vc/include -I/opt/vc/include/interface/vcos/pthreads -I/opt/vc/include/interface/vmcs_host/linux")
   string(CONCAT CMAKE_EXE_LINKER_FLAGS ${CMAKE_EXE_LINKER_FLAGS} " -L/opt/vc/lib")
   string(CONCAT CMAKE_MODULE_LINKER_FLAGS ${CMAKE_MODULE_LINKER_FLAGS} " -L/opt/vc/lib")
-  list(APPEND ffmpeg_conf --enable-rpi --enable-v4l2-request --enable-libdrm --enable-libudev --disable-hwaccel=h264_v4l2request --disable-hwaccel=mpeg2_v4l2request --disable-hwaccel=vp8_v4l2request)
+  list(APPEND ffmpeg_conf --disable-rpi --enable-sand --enable-v4l2-request --enable-libdrm --enable-libudev --disable-hwaccel=h264_v4l2request --disable-hwaccel=mpeg2_v4l2request --disable-hwaccel=vp8_v4l2request)
 #endif()
 
 if(CMAKE_C_FLAGS)
