--- b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/cfg80211.c	2020-06-18 12:01:47.385789393 -0400
+++ c/drivers/net/wireless/broadcom/brcm80211/brcmfmac/cfg80211.c	2020-06-18 12:02:22.103239252 -0400
@@ -5,6 +5,7 @@
 
 /* Toplevel file. Relies on dhd_linux.c to send commands to the dongle. */
 
+#include <linux/version.h>
 #include <linux/kernel.h>
 #include <linux/etherdevice.h>
 #include <linux/module.h>
--- b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c	2020-06-18 12:01:47.382455987 -0400
+++ c/drivers/net/wireless/broadcom/brcm80211/brcmfmac/sdio.c	2020-06-18 13:02:24.463279941 -0400
@@ -3,6 +3,7 @@
  * Copyright (c) 2010 Broadcom Corporation
  */
 
+#include <linux/version.h>
 #include <linux/types.h>
 #include <linux/atomic.h>
 #include <linux/kernel.h>
--- b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/usb.c	2020-06-18 12:01:47.382455987 -0400
+++ c/drivers/net/wireless/broadcom/brcm80211/brcmfmac/usb.c	2020-06-18 13:02:35.666875817 -0400
@@ -3,6 +3,7 @@
  * Copyright (c) 2011 Broadcom Corporation
  */
 
+#include <linux/version.h>
 #include <linux/kernel.h>
 #include <linux/module.h>
 #include <linux/firmware.h>
--- b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/fwsignal.c	2020-06-18 12:01:47.382455987 -0400
+++ c/drivers/net/wireless/broadcom/brcm80211/brcmfmac/fwsignal.c	2020-06-18 13:02:54.510650487 -0400
@@ -2,6 +2,8 @@
 /*
  * Copyright (c) 2010 Broadcom Corporation
  */
+
+#include <linux/version.h>
 #include <linux/types.h>
 #include <linux/module.h>
 #include <linux/if_ether.h>
--- b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/bus.h	2020-06-18 12:01:47.382455987 -0400
+++ c/drivers/net/wireless/broadcom/brcm80211/brcmfmac/bus.h	2020-06-18 13:03:12.981082828 -0400
@@ -3,6 +3,8 @@
  * Copyright (c) 2010 Broadcom Corporation
  */
 
+#include <linux/version.h>
+
 #ifndef BRCMFMAC_BUS_H
 #define BRCMFMAC_BUS_H
 
